packages/invoices/src/OutboundShipment/DetailView/modals/ItemDetailsModal.tsx

```diff
@@ -45,31 +45,42 @@ const listQueryFn = async (): Promise<Item[]> => {
     gql`
       query items {
         items {
-          data {
-            id
-            isVisible
-            name
-            code
-            availableQuantity
-            availableBatches {
-              nodes {
-                id
-                batch
-                expiryDate
-                packSize
-                costPricePerPack
-                sellPricePerPack
-                availableNumberOfPacks
-                totalNumberOfPacks
+          ... on ItemConnector {
+            nodes {
+              id
+              code
+              availableBatches {
+                ... on StockLineConnector {
+                  nodes {
+                    availableNumberOfPacks
+                    batch
+                    costPricePerPack
+                    expiryDate
+                    id
+                    itemId
+                    packSize
+                    sellPricePerPack
+                    storeId
+                    totalNumberOfPacks
+                  }
+                }
+                ... on ConnectorError {
+                  __typename
+                  error {
+                    description
+                  }
+                }
               }
+              isVisible
+              name
             }
           }
         }
       }
     `
   );
 
-  return items.data;
+  return items.nodes;
```

Comment:
As much as graphql wants you to have the data in the shape you want.. it doesn't?